### ISSUE TITLE
Bug: Not all Cameras show up

### ISSUE THREAD

**Issue by merspieler** [you-win/openseeface-gd#87]
I've got the issue on Linux, that only one Camera shows up.

I've got a virtual camera from obs and my webcam.
```
# v4l2-ctl --list-devices
Dummy video device (0x0000) (platform:v4l2loopback-000):
	/dev/video0

Video Capture 4 (usb-0000:0a:00.3-1):
	/dev/video1
	/dev/video2
	/dev/media0
```

In the tracker camera select, only one camera shows up which is the virtual one.
I can tell that cause if the virtual camera in obs is deactivated, I get the following error when I start the tracker:
`[ WARN:0@1271.137] global /io/opencv/modules/videoio/src/cap_v4l.cpp (1000) tryIoctl VIDEOIO(V4L2:/dev/video0): select() timeout.`
While it throws now warnings or errors if it's running.

If you need more information, to track this, please let me know.

**Comment by you-win** [comment]
Thanks for the bug report!

I will need to implement support for multiple cameras on Linux. I've only tested this on a laptop running Fedora that had a builtin webcam.

**Comment by you-win** [comment]
Hopefully fixed by #98

**Comment by merspieler** [comment]
thanks, I'll check it out.
